cinema_service/urls.py

```diff
@@ -5,4 +5,6 @@
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("__debug__/", include("debug_toolbar.urls")),
+
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Similar to the previous comment, ensure that `user.urls` has an `app_name` defined if you are using the `namespace` argument in the `include` function.